src/XrdCl/XrdClZipArchive.cc

```diff
@@ -321,10 +321,17 @@ namespace XrdCl
                                                             "End-of-central-directory signature not found." );
                                         Pipeline::Stop( error );
                                       }
+                                      try{
```

Comment:
I forgot to pass the maxSize argument, and inserted it now as the length of the chunk we read minus the difference between the chunk start and eocd start, since this should give exactly the remaining mapped memory length. uint32_t is sufficient because the maximum reading length is just more than 65000.